.github/workflows/cd.yml

```diff
@@ -43,5 +50,5 @@ jobs:
           draft: false
           prerelease: false
           token: ${{ secrets.GITHUB_TOKEN }}
-          artifacts: './dist/system.json,./dist/pf2e.zip'
+          artifacts: "./dist/system.json,./dist/pf2e.zip,./packs-json/packs-json.zip"
```

Comment:
I seem to remember github actions being picky about quote style. Did you run your test before or after changing this?

Reply:
I've set up prettier to run on save so that would have been changed for all my tests but I can change it back to single quotes to be safe.

Reply:
eh, let's try and see